checker/match.go

```diff
@@ -174,3 +177,47 @@ func matchHTTPMethods(methods []string, reqMethod string) bool {
 	log.Debug("HTTP Method not matched.")
 	return false
 }
+
+func matchL4Protocol(rule *proto.Rule, dest *authz.AttributeContext_Peer) bool {
+	// Extract L4 protocol type of socket address for destination peer context. Match against rules.
```

Comment:
drop a Debug log like the other match functions do.

Reply:
Yes. Done.